finagle/h2/src/test/scala/com/twitter/finagle/buoyant/h2/netty4/Netty4StreamTransportTest.scala

```diff
@@ -78,6 +78,7 @@ class Netty4StreamTransportTest extends FunSuite {
 
       eventually { assert(transport.onReset.isDefined) }
       assert(await(transport.onReset.liftToTry) == Throw(StreamError.Remote(rst)))
+      ()
     }
```

Comment:
It returns [Assertion](http://doc.scalatest.org/3.0.1/index.html#org.scalatest.compatible.Assertion) which is just a marker trait.